macOS/DuckDuckGo/NavigationBar/View/AddressBarButtonsViewController.swift

```diff
@@ -281,6 +290,16 @@ final class AddressBarButtonsViewController: NSViewController {
         openPrivacyDashboardPopover()
     }
 
+    @IBAction func aiChatButtonAction(_ sender: Any) {
+        let shouldOpenInNewTab = NSEvent.modifierFlags.contains(.command)
```

Comment:
I understand that this is a product decision, but perhaps worth raising that cmd+click in other places of the app opens a new tab in background, not foreground. See `NewTabPageLinkOpener.open()` (available in main) for the full logic (also including shift and option key modifiers).

Reply:
I’ll do it. I think it makes sense. Thanks